### ISSUE TITLE
Error in man page building

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1170]
```
building [man]: all manpages
updating environment: 0 added, 0 changed, 0 removed
looking for now-outdated files... none found
writing... predicatesolver.1 { discussion implementation } 
Exception occurred:
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/docutils/writers/manpage.py", line 860, in indent
    step = self._indent[-1]
IndexError: list index out of range
The full traceback has been saved in /var/folders/rg/h22wr0j9483f22nbbhwk6lhc0000gt/T/sphinx-err-CY3sGL.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
Either send bugs to the mailing list at <http://groups.google.com/group/sphinx-dev/>,
or report them in the tracker at <https://bitbucket.org/birkenfeld/sphinx/issues/>. Thanks!
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1170
- Originally reported by: Victor Eijkhout
- Originally created at: 2013-05-13T15:37:19.990

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2013-07-26 01:51:02+00:00_

Can you attach your rst file (or smallest example that can reproduce the issue) ?

This issue might have been fixed in the Sphinx-1.2b1.

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2013-07-26 15:03:56+00:00_

[Victor Eijkhout](https://bitbucket.org/VictorEijkhout) I couldn't reproduce the problem with both 1.1.3 and 1.2b1.
I’m suspicious that `ig.py` contains the cause of the problem.

Could you create a minimal example to reproduce the problem?
Thanks.
